setup.py

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-NAME = 'pyscf-forge'
+NAME = 'pyscf_forge'
```

Comment:
This won't oblige us to rename the repository itself, will it?

Reply:
This change only affects how this project shows on PyPI index page. When using pip, `pip install pyscf-forge` is equivalent to `pip install pyscf_forge`. Pip can do the conversion automatically. I don't it's necessary to change the repository name.